homeassistant/components/todoist/calendar.py

```diff
@@ -421,7 +421,7 @@ def create_todoist_task(self, data):
                 # it shouldn't be counted.
                 return None
 
-            task[DUE_TODAY] = task[END].date() == datetime.today().date()
+            task[DUE_TODAY] = task[END].date() == dt.utcnow().date()
```

Comment:
Per the [documentation](https://docs.python.org/3/library/datetime.html#datetime.datetime.today), `datetime.today` returns the current datetime in the local timezone.  This is fine if your local timezone is UTC, but if it's anything else, this check would fail in cases where the UTC date was different that the local timezone date.  `task[END]` is always UTC.